planning/behavior_velocity_planner/src/scene_module/blind_spot/scene.cpp

```diff
@@ -309,6 +313,10 @@ int BlindSpotModule::insertPoint(
     // copy from previous point
     inserted_point = inout_path->points.at(std::max(insert_idx - 1, 0));
     inserted_point.point.pose = path_ip.points[insert_idx_ip].point.pose;
+    if (planning_utils::calcDist2d(inserted_point, inout_path->points.at(insert_idx).point) < 0.1) {
```

Comment:
I'm not sure about the meaning of 0.1, but is 0.1 small enough?

Reply:
If the length between inserted stop point here and the actual stop line will be 0.1 at maximum, I think we'd better make 0.1 smaller.

Reply:
constepr float min_distance = 0.1

Reply:
changed to 0.001 at [7581a25](https://github.com/autowarefoundation/autoware.universe/pull/505/commits/7581a2531da9fac304dd64c920d82bfb184c39d1)

Reply:
pls create variable
```
constexpr double min_dist = 0.001;
```